content/sql/concepts/window-functions/terms/lag/lag.md

```diff
@@ -59,3 +59,41 @@ Output
 | jenna      | black     | 35  | 60           |
 | chris      | smith     | 30  | 35           |
 | dave       | james     | 19  | 30           |
+
+## Example 
+### Using partition by
+
+This example demonstrates how to use the `LAG()` function to create a new column, `previous_position`. The function retrieves the value from the previous row within the same `employee_id` group in the `Promotions` table. This is achieved by using the `PARTITION BY employee_id` clause.
```

Comment:
The explanation for the `previous_position` column in the second example could confuse readers unfamiliar with SQL's `PARTITION BY`. It can be better explained as follows:

```suggestion
This example demonstrates how to use the `LAG()` function to create a new column, `previous_position`. 

The `PARTITION BY employee_id` clause ensures that the `LAG()` function operates within each group of rows that share the same `employee_id`. The `ORDER BY promotion_date` ensures the rows are processed in chronological order.
```